x-pack/plugins/ml/public/application/explorer/explorer_charts/components/explorer_chart_label/explorer_chart_label.js

```diff
@@ -48,16 +49,30 @@ export function ExplorerChartLabel({
   const entityFieldBadges = entityFields.map((entity) => {
     const key = `${infoTooltip.chartFunction}-${entity.fieldName}-${entity.fieldType}-${entity.fieldValue}`;
     return (
-      <Fragment key={`badge-wrapper-${key}`}>
-        <ExplorerChartLabelBadge entity={entity} />
+      <EuiFlexGroup gutterSize="none" alignItems="center" key={`badge-wrapper-${key}`}>
```

Comment:
Are the changes here causing the badges in the Anomaly Explorer charts to be rendered with line breaks after the detector description label and before the info tooltip?

Before:
<img width="635" alt="Screenshot 2024-11-11 at 10 29 34" src="https://github.com/user-attachments/assets/d162e5d3-8948-45dd-8fa1-bbb8a3965221">

After:
<img width="1605" alt="Screenshot 2024-11-11 at 10 16 45" src="https://github.com/user-attachments/assets/dca34d03-e637-470d-a849-18520c9e73a5">

Can the line breaks be removed? (Could also remove the `-` character before the badge - not sure that adds anything.

Reply:
Okay, I went ahead and reverted most of the changes to this layout since it relies on some css and made the least amount of changes possible. 21cb885aa59584bdad41feac4f2f1a76f412ce2d
We should update this to use components in a separate PR.